azkaban-common/src/main/java/azkaban/sla/SlaOption.java

```diff
@@ -302,23 +313,26 @@ public String createSlaMessage(final ExecutableFlow flow) {
    * @return the job level SLA options.
    */
   static public List<SlaOption> getJobLevelSLAOptions(List<SlaOption> options) {
-    return options.stream().filter(rule -> rule.type.getComponent() == SlaType.ComponentType.JOB).collect
-        (Collectors.toList());
+    return options.stream().filter(rule -> rule.type.getComponent() == SlaType.ComponentType.JOB)
+        .collect
+            (Collectors.toList());
   }
 
   /**
    * @param options a list of SLA options.
    * @return the flow level SLA options.
    */
   static public List<SlaOption> getFlowLevelSLAOptions(List<SlaOption> options) {
-    return options.stream().filter(rule -> rule.type.getComponent() == SlaType.ComponentType.FLOW).collect
-        (Collectors.toList());
+    return options.stream().filter(rule -> rule.type.getComponent() == SlaType.ComponentType.FLOW)
+        .collect
+            (Collectors.toList());
```

Comment:
Could 327 and 328 be condensed to 1 line.

Reply:
sure. will address